src/Core/Framework/DataAbstractionLayer/EntityRepository.php

```diff
@@ -86,7 +86,7 @@ public function __construct(
      */
     public function setEntityLoadedEventFactory(EntityLoadedEventFactory $eventFactory): void
     {
-        if ($this->eventFactory !== null) {
+        if (!isset($this->eventFactory)) {
```

Comment:
you flipped the IF

`if (isset($this->eventFactory)) { ... }`

Reply:
Oops, you are right, it was too late after all, and I was thinking if this should be also covered in a test, but was unsure how (if one really wants to do reflection in that case).

Anyway the if should be fixed now :-)